src/runtime-rs/crates/hypervisor/src/ch/inner_hypervisor.rs

```diff
@@ -1333,4 +1384,50 @@ mod tests {
             assert_eq!(d.level, level, "{}", msg);
         }
     }
+
+    #[actix_rt::test]
+    async fn test_get_thread_ids() {
+        #[derive(Debug)]
+        struct TestData<'a> {
+            proc_path: &'a str,
+            result: Result<HashMap<u32, u32>>,
+        }
+
+        let tests = &[
+            TestData {
+                proc_path: "",
+                result: Err(anyhow!("proc path must be non-empty strings.")),
+            },
+            TestData {
+                proc_path: "/tmp/12345.6",
```

Comment:
The correction was made for the case when the passed path does not corresponds to a valid `proc/pid` path i.e. `/usr/lib/os-release.

However, replacing the `/proc/1` path was avoided because in this test case we want to validate two conditions: that `proc/${path}` exists and that the pid does not belong to `ch`.